generators/aws-containers/templates/_base.template.yml

```diff
@@ -0,0 +1,35 @@
+AWSTemplateFormatVersion: 2010-09-09
+Parameters:
+  applicationStackS3Bucket:
+    Type: String
+    Description: Name of the S3 bucket containing the application.template.yml template file
+    Default: <%= aws.s3BucketName %>
+<% for (app of appConfigs) { %>  <%= app.baseName %>DBPassword:
+    Type: String
+    Description: Aurora master database password for <%= app.baseName %>
+    NoEcho: true
+<%} %>
+  shouldDeployService:
+    Type: String
+    Description: Should the ECS service be initialised
+    AllowedValues:
+        - true
+        - false
+    Default: false
+Resources:
+<% for (app of appConfigs) { %>  <%= app.baseName %>:
+    Type: 'AWS::CloudFormation::Stack'
+    Properties:
+      Parameters:
+        parentStackName: !Ref AWS::StackName
+        repositoryName: <%= app.baseName %>
```

Comment:
@ggotti You can't have uppercase in the repository name. Needs to comply with the following regex:
```
 '(?:[a-z0-9]+(?:[._-][a-z0-9]+)*/)*[a-z0-9]+(?:[._-][a-z0-9]+)*'
```